File: ver_py/T15.py
# -*- coding: utf-8 -*-
# https://leetcode.cn/problems/determine-color-of-a-chessboard-square/
class Solution:
    def squareIsWhite(self, coordinates: str) -> bool:
        x = ord(coordinates[0]) - ord('a')
        y = int(coordinates[1]) - ord("1")
        return (x + y) % 2 == 1
    
if __name__ == "__main__":
    s = Solution()
    coordinates = input()
    print(s.squareIsWhite(coordinates))

File: ver_cpp/T15.cpp
// https://leetcode.cn/problems/determine-color-of-a-chessboard-square/
#include<bits/stdc++.h>
using namespace std;

class Solution {
public:
    bool squareIsWhite(string coordinates) {
        int x = coordinates[0] - 'a';
        int y = coordinates[1] - '1';
        return (x + y) % 2 == 1;
    }
};

int main(){
    Solution s;
    string coordinates;
    cin>>coordinates;
    cout<<s.squareIsWhite(coordinates)<<endl;
    return 0;
}